applications/forms.py

```diff
@@ -80,7 +80,7 @@ def clean_code_conduct(self):
         # self.instance.pk is None if there's no Application existing before
         # https://stackoverflow.com/questions/9704067/test-if-django-modelform-has-instance
         if not cc and not self.instance.pk:
-            raise forms.ValidationError("In order to apply and attend you have to accept our code of conduct")
+            raise forms.ValidationError("You must accept our code of conduct")
```

Comment:
Any specific reason to change the language here? I think "must" sounds a bit tough 😅

Reply:
How about `To attend %s you must abide by our code of conduct` instead? Still uses `must` but I think lightened a little by the context?

Reply:
Sounds better 👍